contrib/utilities/checkdoxygen.py

```diff
@@ -49,6 +49,37 @@ def check_empty_lines_in_tables(lines):
 
     return
 
+# have more than one header with the same name in a tutorial?
+def check_multiple_defined_headers(lines):
+    headers = []
+    for l in lines:
+        # this may break if a header splits across a line
+        if "<h" in l:
+            # convert thisheader to the doxygen header name equivalent
+            thisheader = l                             # take this line
+            thisheader = thisheader.replace(' ', '')   # remove all whitespace
+            thisheader = thisheader.split('<h')[1][2:] # remove open header tag
+            thisheader = thisheader.split('</h')[0]    # remove close header tag
+            thisheader = ''.join(ch for ch in thisheader if ch.isalnum()) # remove nonalphanumeric
+
+            if thisheader in headers:
+                sys.exit("Error: repeated header title '%s' in file group '%s'"%(thisheader,args[0]))
+            else:
+                headers.append(thisheader)
+        
+        if "@sect" in l:
+            thisheader = l                             # take this line
```

Comment:
please add a comment ``detect @sect[1-4]{...}``

Reply:
Got it!